indexer/rank_table.cpp

```diff
@@ -217,7 +222,23 @@ unique_ptr<RankTable> LoadRankTable(unique_ptr<TRegion> && region)
 // Calculates search rank for a feature.
 uint8_t CalcSearchRank(FeatureType const & ft)
 {
-  return feature::PopulationToRank(ftypes::GetPopulation(ft));
+  uint8_t eventRank = 0;
+  Classificator const & c = classif();
+  auto const types = feature::TypesHolder(ft);
+  auto const fcType =
+      ftypes::BaseChecker::PrepareToMatch(c.GetTypeByPath({"event", "fc2018"}), 2 /* level */);
+  auto const fcCityType =
+      ftypes::BaseChecker::PrepareToMatch(c.GetTypeByPath({"event", "fc2018_city"}), 2 /* level */);
+  if (find(types.begin(), types.end(), fcType) != types.end() ||
+      find(types.begin(), types.end(), fcCityType) != types.end())
+  {
+    eventRank = 16;  // todo(@t.yan): test and adjust.
+  }
```

Comment:
Probably factor this out to a separate function? CalcEventRank?

Reply:
done